backend/src/feature-flags/isFeatureEnabled.ts

```diff
@@ -0,0 +1,18 @@
+import { PostHog } from 'posthog-node'
+import { API_CONFIG } from '../config'
+import { FeatureFlag, Edition } from '../types/common'
+
+export default async (
+  featureFlag: FeatureFlag,
+  tenantId: string,
+  posthog: PostHog,
+): Promise<boolean> => {
+  if (API_CONFIG.edition === Edition.COMMUNITY) {
+    return true
+  }
+
+  const featureFlagEnabled = await posthog.isFeatureEnabled(featureFlag, '', {
```

Comment:
Can certain feature flags be cached? Like for an hour or a day or depending on the conditions behind it? I imagine this being used in almost CRUD all API requests and could be slowing things down.

Reply:
Posthog is already caching it to memory, so consequent isFeatureEnabled calls will be retrieved from memory.
However I don't think there's an easy way to use this effectively because of multiple api pods.
Any ideas?